library/std/src/os/mod.rs

```diff
@@ -135,6 +135,9 @@ pub mod macos;
 pub mod netbsd;
 #[cfg(target_os = "openbsd")]
 pub mod openbsd;
+#[cfg(target_os = "psp")]
+#[unstable(feature = "psp_std", issue = "none")]
```

Comment:
Is `#[unstable]` required? The PSP is a tier 3 target so I think we don't need this at all.

Reply:
_Some_ stability attribute is required for all `std` modules/functions/... Marking it as stable feels wrong.

Reply:
But I can move it into `psp/mod.rs`

Reply:
Oh I see, well if it's required then this LGTM. Thanks!